scripts/scil_compute_bundle_profiles.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Compute bundle profiles and their statistics along streamlines.
+"""
+
+
+import argparse
+import json
+import os
+
+import nibabel as nib
+import numpy as np
+
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.image import assert_same_resolution
+from scilpy.io.utils import (assert_inputs_exist,
+                             add_json_args,
+                             add_overwrite_arg,
+                             add_reference_arg)
+from scilpy.utils.filenames import split_name_with_nii
+from scilpy.utils.metrics_tools import get_bundle_metrics_profiles
+from scilpy.tracking.tools import resample_streamlines_num_points
+from scilpy.tractanalysis.features import get_streamlines_centroid
+
+
+def norm_l2(x):
+    return np.sqrt(np.sum(np.power(x, 2), axis=1, dtype=np.float32))
+
+
+def average_euclidean(x, y):
+    return np.mean(norm_l2(x - y))
+
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                description=__doc__)
+    p.add_argument('in_bundle',
+                   help='Fiber bundle file to compute the bundle profiles on.')
```

Comment:
I don't like that sentence, I would prefer something more simple _Input bundle for profiling._